partner_vat_unique/models/res_partner.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 Grant Thornton Spain - Ismael Calvo <ismael.calvo@es.gt.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools import config
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    vat = fields.Char(copy=False)
+
+    @api.constrains("vat")
+    def _check_vat_unique(self):
```

Comment:
In V13 there's this method [`def _compute_same_vat_partner_id(self)`](https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/models/res_partner.py#L261-L269) that looks quite similar.

Reply:
Now used the field "same_vat_partner_id", thanks.